winlogbeat/winlogbeat.reference.yml

```diff
@@ -51,12 +51,22 @@ winlogbeat.event_logs:
 # sub-dictionary. Default is false.
 #fields_under_root: false
 
-# Internal queue size for single events in processing pipeline
-#queue_size: 1000
-
-# The internal queue size for bulk events in the processing pipeline.
-# Do not modify this value.
-#bulk_queue_size: 0
+# Internal queue configuration for buffering events to be published.
+#queue:
```

Comment:
I'll open a PR to delete this [block of code](https://github.com/elastic/beats/blob/09e90c66c1a1aa7e5301e54ff61219af21e705f1/winlogbeat/config/config.go#L43-L62) from Winlogbeat so that `queue` isn't rejected.